paper-toast.html

```diff
@@ -193,6 +197,19 @@
         },
 
         /**
+         * Called on transitions of the toast, indicating a finished animation
+         *
+         * @private
+         */
+        _finishTransitioning: function(e) {
+          if (e && e.propertyName === "opacity" && this.opened) {
```

Comment:
Maybe add a comment here explaining why we check for `propertyName`. Maybe a check like this is "cleaner":
```javascript
if (event && event.target === this && event.propertyName == 'opacity') {
  if (this.opened) {
    this._finishRenderOpened();
  } else {
    this._finishRenderClosed();
  }
}